app/templates/views/guidance/using-notify/delivery-times.html

```diff
@@ -34,9 +34,23 @@ <h2 id="letters" class="heading-medium">Letters</h2>
   <p class="govuk-body">First class letters are delivered one day after they’re dispatched. Second class letters are delivered 2 days after they’re dispatched. Royal Mail delivers from Monday to Saturday, excluding bank holidays.</p>
   <p class="govuk-body">Letters to Europe are delivered 3 to 5 days after they’re dispatched. Letters sent anywhere else in the world take 5 to 7 days to arrive.</p>
 
-  <h3 id="returned-mail" class="heading-small">Undelivered letters and returned mail</h3>
-  <p class="govuk-body">Every letter we send includes our printer’s address on the back of the envelope. It’s not possible to customise the return address.</p>
-  <p class="govuk-body">Returned mail is destroyed by our printer.</p>
-  <p class="govuk-body">Each letter has a unique ID. Our printer sends us the ID of every letter they destroy. We’ll contact you if the IDs match any of your letters. This is a partly manual process, so it can take a few weeks.</p>
+  <h2 id="returned-mail" class="heading-medium">Returned letters</h2>
+  <p class="govuk-body">Every letter we send includes our print provider’s address on the back of the envelope. You cannot customise the return address.</p>
+  <p class="govuk-body">Undelivered letters are returned to our print provider and destroyed.</p>
+  <p class="govuk-body">You can see a report of returned letters on the dashboard.</p>
+  <h3 id="returned-mail" class="heading-small">Identify the returned letters</h3>
+ <p class="govuk-body">To help you identify what letters are returned, the report will show details such as the:
+ </p>
+ <ul class="govuk-list govuk-list--bullet">
+   <li>template ID</li>
+   <li>date the letter was sent</li>
+   <li>a reference for each letter if you include this in the list of addresses you upload</li>
```

Comment:
I think we should include a bit more about the reference here because it’s not specific to letters that are uploaded 

```suggestion
   <li>a reference for each letter if you include either:
       <ul class="govuk-list govuk-list--bullet">
           <li>a column called ‘reference’ in the list of addresses you upload</li>
           <li>the <code>reference</code> argument when making an API call</li>
       </ul>
   </li>
```